src/playground-project.ts

```diff
@@ -78,29 +78,42 @@ export class PlaygroundProject extends LitElement {
   }
 
   /**
-   * Get or set the array of project files.
+   * Get or set the project config.
    *
-   * When both `projectSrc` and `files` are set, the one set most recently wins.
-   * Slotted children win only if both `projectSrc` and `files` are undefined
+   * When both `projectSrc` and `config` are set, the one set most recently
+   * wins. Slotted children win only if both `projectSrc` and `config` are
+   * undefined.
    */
   @property({attribute: false, hasChanged: () => false})
```

Comment:
Can be on getter or setter, doesn't matter which. I moved it to getter, maybe easier to read.

Expanded comment.

Reply:
> LGTM. Change seems ok. Why did it not affect tests?

Because there wasn't a test that covered setting the `files` property. Added a new one that covers the new `config` property.